### ISSUE TITLE
Logs out on some websites after browser restart

### ISSUE THREAD

**Issue by maximbaz** [qutebrowser/qutebrowser#5782]
**Version info**:
```
qutebrowser v1.13.1
Git commit: f0f15b89b on master (2020-10-09 15:11:53 +0200)
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.1
```

**Does the bug happen if you start with `--temp-basedir`?**: Yes

**Description**

On some websites qutebrowser doesn't preserve the session, and every time I restart browser I am presented with a login page. I'm not sure if this is per-site issue, but I have a few, most are in corpnet but one is in public internet which I can luckily share with you.

**How to reproduce**

1. `git pull; python3 scripts/mkvenv.py`
1. `.venv/bin/qutebrowser --basedir /tmp/qute123 -s auto_save.session true`
1. open https://demo.sogo.nu/SOGo/ and use `sogo1` as both username and password to login
1. `:q`
1. `.venv/bin/qutebrowser --basedir /tmp/qute123 -s auto_save.session true`

Expected: you are still logged in
Actual: you are presented with a login page

If I try this in Chromium, I am still logged in after browser restart.

**Comment by toofar** [comment]
TL;DR: Known issue, no fix planned.
I assume when you try it in chromium you have selected "remember where I left off" or whatever? When I log in with those credentials (and remember my username) there are three cookies. Deleting them one by one and reloading the 0xHIGHFLYxSoGo (why!?) one seems to be the one that indicates you are logged in. It is marked as a session cookie, meaning it should be deleted when the browser shuts down, or as the modern interpretation says, when your browsing session ends. Modern browsers let you extend the idea of a browsing session across browser restarts. QTWebengine doesn't have this concept, nor does qutebrowser. So the cookie is cleared when you restart.

**Comment by maximbaz** [comment]
Thank you for the explanation, I had no idea, I completely agree with how it is handled by QTWebEngine / qutebrowser.
